webapp/src/main/java/life/qbic/views/login/LoginHandler.java

```diff
@@ -19,15 +19,20 @@
 @Component
 public class LoginHandler implements LoginHandlerInterface, ConfirmEmailOutput {
 
-  private final UserJpaRepository userRepository;
-
   private LoginLayout registeredLoginView;
 
   private final ConfirmEmailInput confirmEmailInput;
+  private static final ErrorMessage INCORRECT_USERNAME_OR_PASSWORD = new ErrorMessage(
+      "Incorrect username or password",
+      "Check that you have entered the correct username and password and try again."
+  );
+
+  private static final InformationMessage EMAIL_CONFIRMATION_SUCCESS = new InformationMessage(
+      "Email address confirmed",
+      "You can now login with your credentials."
+  );
```

Comment:
NIT: The creation of the message component could be handled by the view.

Reply:
As the handler decides which message is shown and added/removed, it belongs in the handler imho.